export.js

```diff
@@ -15,6 +16,9 @@ function commonExportTests(exportFn, expectedFn, expectedErrorsFn) {
         errors: expectedErrorsFn(name)
       };
     } catch (e) {
+      if (e instanceof AssertionError) {
```

Comment:
Looking at the `AssertionError` source code [here](https://github.com/chaijs/assertion-error/blob/master/index.js#L81-L91), I think we can avoid pulling in `AssertionError` as a dependency at all by doing this:

```js
if (e instanceof Error && e.name === 'AssertionError')
```

This way we could avoid needing to declare it as a peer dependency (or needing to declare it at all).

Reply:
That's true. I think that the only way things could go wrong is if an exporter threw a different `AssertionError` as part of loading a fixture and did not intend it to be a test failure. That seems pretty unlikely.

The only other advantage of declaring it as a peer dependency is that we'd get to experiment with how they behave before mucking about with peer deps for all of shr-models.

Reply:
I'm not too concerned about some other different `AssertionError`.  I think we'd want the error to pop up no matter what anyway.

As for trying out peer dependencies... nah.  I think that when we have a chance to try them w/ shr-models, we should be able to fully test them OK without needing a pre-experiment.

Does that sound OK to you?